apps/davi/src/stores/modules/1_5/checkDataSourceAvailability.ts

```diff
@@ -1,8 +1,48 @@
 import { subgraphClientsUris } from 'clients/apollo';
 import { SupportedSubgraph } from 'stores/types';
 
-// TODO: Check for health of connection if found
+export const checkDataSourceAvailability = async chainId => {
+  try {
+    if (!!subgraphClientsUris?.[chainId]?.[SupportedSubgraph.Governance1_5]) {
+      console.debug('No subgraph URL for this chain, using fallback');
+      return false;
+    }
 
-export const checkDataSourceAvailability = chainId => {
-  return !!subgraphClientsUris?.[chainId]?.[SupportedSubgraph.Governance1_5];
+    const query = `{
+    _meta {
+      block {
+        number
+        timestamp
+      }
+    }
+   }`;
+
+    const response = await fetch(subgraphClientsUris[chainId]['Guilds'], {

```

Comment:
Be aware of merging develop to this branch as after https://github.com/DXgovernance/davi-monorepo/pull/251 was merged `['Guilds']` should change to `SupportedSubgraph.Guilds`